streaming/base/storage/upload.py

```diff
@@ -336,13 +394,16 @@ class OCIUploader(CloudUploader):
             a remote location. Default to ``False``.
     """
 
-    def __init__(self,
-                 out: Union[str, Tuple[str, str]],
-                 keep_local: bool = False,
-                 progress_bar: bool = False) -> None:
+    def __init__(
+        self,
+        out: Union[str, Tuple[str, str]],
+        keep_local: bool = False,
+        progress_bar: bool = False,
```

Comment:
Do you mind reverting the trailing comma `,` changes as this is not related to this PR? Thanks!